gef.py

```diff
@@ -453,16 +453,18 @@ def gef_print(x="", less=False, *args, **kwargs):
     """Wrapper around print(), using string buffering feature."""
     x = HighlightCommand.highlight_text(x)
 
+    if not x:
+        return
+
```

Comment:
There is a call like `gef_print("")` in gef source.
This is used when you do not want to use Pager and simply want to include a line break.
If you check `not x` here, the expected behavior will not occur.

Reply:
Reverted!